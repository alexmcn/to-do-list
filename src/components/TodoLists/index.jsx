import React from 'react';

import { makeStyles } from '@material-ui/core/styles';
import { List, ListItem, ListItemIcon, ListItemText } from '@material-ui/core';
import ListIcon from '@material-ui/icons/List';

const useStyles = makeStyles(theme => ({
  root: {
    width: '100%',
    maxWidth: 360,
  },
}));

const TodoLists = () => {
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <List component="nav" aria-label="Main Lists">
        <ListItem button>
          <ListItemIcon>
            <ListIcon />
          </ListItemIcon>
          <ListItemText primary="Team To-Do List" />
        </ListItem>
      </List>
    </div>
  );
};

export default TodoLists;
