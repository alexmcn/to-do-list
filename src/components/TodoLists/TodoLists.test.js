import React from 'react';
import renderer from 'react-test-renderer';

import TodoLists from '.';

describe('TodoLists', () => {
  it('has area containing a list of todo lists with 1 fixed list titled "Team To-Do List"', () => {
    const instance = renderer.create(<TodoLists />);

    expect(instance.toJSON()).toMatchSnapshot();
  });
});
