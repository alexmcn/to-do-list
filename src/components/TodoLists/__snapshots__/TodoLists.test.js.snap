// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoLists has area containing a list of todo lists with 1 fixed list titled "Team To-Do List" 1`] = `
<div
  className="makeStyles-root-1"
>
  <nav
    aria-label="Main Lists"
    className="MuiList-root MuiList-padding"
  >
    <div
      aria-disabled={false}
      className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex={0}
    >
      <div
        className="MuiListItemIcon-root"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M3 13h2v-2H3v2zm0 4h2v-2H3v2zm0-8h2V7H3v2zm4 4h14v-2H7v2zm0 4h14v-2H7v2zM7 7v2h14V7H7z"
          />
        </svg>
      </div>
      <div
        className="MuiListItemText-root"
      >
        <span
          className="MuiTypography-root MuiListItemText-primary MuiTypography-body1"
        >
          Team To-Do List
        </span>
      </div>
    </div>
  </nav>
</div>
`;
